### PR TITLE
Allow to pass options to reshape-minify package

### PR DESCRIPTION
**Pull request by liitfr**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "reshape-standard",
   "description": "a standard plugin pack for reshape",
-  "version": "3.0.1",
+  "version": "3.0.2",
```
Comment (jescalan) [inline package.json]: So typically the maintainer will handle the version bumps, you don't need to include this in the pull request

**Comment by liitfr** [comment]
oups sorry for that ! reverted last commit, should be fine now !

**Comment by jescalan** [comment]
Perfect! I'll get this out right now. Thanks for this great contribution 😁

**Comment by liitfr** [comment]
that was just 10 lines of code ^^ can I ask you how you deal with this versioning process you were talking about ? is it a tool to automate this process ? just curious ...

**Comment by jescalan** [comment]
Hey it makes a difference though!

Yeah, I use [np](https://github.com/sindresorhus/np), it is truly wonderful

**Pull request merged**